tests/test_validator.py

```diff
@@ -115,13 +116,18 @@ def setUpClass(cls):
             for i in range(2)])
 
 
-class TestBaseValidator(TestValidatorTemplate):
+class TestBaseValidator(_TestValidatorBase):
     @classmethod
     def setUpClass(cls):
         cls.validator = _TaskValidator(task_type=TaskType.classification,
             few_samples_thr=1, imbalance_ratio_thr=50, far_from_mean_thr=5.0,
             dominance_ratio_thr=0.8, topk_bins=0.1)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_has_enum_entries_in_environment(self):
+        for key in TaskType._member_names_:
```

Comment:
```suggestion
        for key in TaskType.__members__:
```

`_member_names_` is undocumented.

Reply:
Changed